src/main/kotlin/org/jitsi/nlj/rtcp/RetransmissionRequester.kt

```diff
@@ -155,11 +154,9 @@ class RetransmissionRequester(
                         // The work is now due either sooner or later than we previously thought, so
                         // re-schedule the task
                         currentTaskHandle?.cancel(false)
-                        currentTaskHandle = scheduler.schedule(
-                                ::doWork,
-                            Duration.between(clock.instant(), newWorkDueTs).toMillis(),
-                            TimeUnit.MILLISECONDS
-                        )
+                        currentTaskHandle = scheduler.unsafeSchedule(
```

Comment:
Are you thinking just change all the existing to `unsafeXXX` for now and do the cleanup later?

Reply:
Yes, if we agree on the general approach I can do it in this PR, too.

Reply:
I'm fine with splitting it up to keep the initial PRs simple